exercises/complex-numbers/complex_numbers_test.cpp

```diff
@@ -0,0 +1,218 @@
+#include "complex_numbers.h"
+
+#include <cmath>
+#include <complex>
+
+#include "test/catch.hpp"
+
+// Define M_PI for platforms that don't include it
+#ifndef M_PI
+#define M_PI 3.14159265358979323846
+#endif
+
+// Complex Numbers exercise test case data version 1.0.0
+
+using complex_numbers::Complex;
+
+TEST_CASE("add_purely_real_numbers") {
```

Comment:
[problem-specifications #1140](https://github.com/exercism/problem-specifications/pull/1140) has since reordered the test suite with a better workflow for students. Instead of jumping straight to addition they first test real and imaginary parts, then move to more complex operations.

Reply:
Updated everything to the latest test suite, which reordered all the tests and added an extra at the end.